### PR TITLE
Updated Google.Chrome to version 92.0.4515.107

### PR DESCRIPTION
**Pull request by ImJoakim**
image

```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget install -m "C:\Users\WDAGUtilityAccount\Desktop\manifests\g\Google\Chrome\92.0.4515.107"
Found Google Chrome [Google.Chrome]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise64.msi
  ██████████████████████████████  76.2 MB / 76.2 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

image

```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget install -m "C:\Users\WDAGUtilityAccount\Desktop\manifests\g\Google\Chrome\92.0.4515.107"
Found Google Chrome [Google.Chrome]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.google.com/edgedl/chrome/install/GoogleChromeStandaloneEnterprise.msi
  ██████████████████████████████  73.5 MB / 73.5 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21946)

### CODE REVIEW

File: manifests/g/Google/Chrome/92.0.4515.107/Google.Chrome.installer.yaml
Comment (vedantmgoyal2009) [inline manifests/g/Google/Chrome/92.0.4515.107/Google.Chrome.installer.yaml:30]:
```suggestion
  InstallerSha256: DA69FBAEDFBFD021F902331F68646E55740E1A0AE535A9C4570ABB583C12DAE2
```

**Review by vedantmgoyal2009 — CHANGES_REQUESTED** [review]

**Review by ImJoakim — COMMENTED** [review]
Oh, the suggestion was deleted so I was forced to "review" because I tried to comment while it was deleted.

**Comment by ImJoakim** [comment]
Need to wait a little and re-run it.

**Comment by ImJoakim** [comment]
Heya @OfficialEsco, do you know anything about a change made to when the bot is running `/AzurePipelines run`? It doesn't seem to run that command/comment after close -> open anymore.

**Comment by OfficialEsco** [comment]
No idea, seems to act very weird since they got back to work this week, hopefully its a side effect of them being in the middle of something, and didn't fix it before leaving yesterday..

It also seems like its just the msges that are delayed cuz i get a buuuunch of messages at the same time, like i got 30+ mails right now in a 1min timespan lmao

**Comment by ImJoakim** [comment]
> No idea, seems to act very weird since they got back to work this week, hopefully its a side effect of them being in the middle of something, and didn't fix it before leaving yesterday..
> 
> It also seems like its just the msges that are delayed cuz i get a buuuunch of messages at the same time, like i got 30+ mails right now in a 1min timespan lmao

Yeah, it's acting really unusual.

**Comment by OfficialEsco** [comment]
Ahh nvm, there is no schedule, the bot just pick random PR's, https://github.com/microsoft/winget-pkgs/pull/21949#issuecomment-884134661 have been idle for 2 hours while a bunch of my PR's have gone trough

**Comment by ImJoakim** [comment]
Oof

**Comment by ImJoakim** [comment]
Whoops, that was not the **Comment** button haha

**Comment by vedantmgoyal2009** [comment]
There are doing changes to validation merge.
Issue: https://github.com/microsoft/winget-pkgs/issues/21008
Milestone: https://github.com/microsoft/winget-pkgs/milestone/11?closed=1

**Comment by OfficialEsco** [comment]
That is just so us Moderators can Approve before its Validated, has nothing to do with this

**Comment by vedantmgoyal2009** [comment]
The bot forgets to merge because there is something that has not been fixed yet.

**Comment by vedantmgoyal2009** [comment]
I don't know correctly, but I think they have made changes in msftbot and wingetbot due to which wingetbot forgetsbto comment and msftbot does not merge it instantly once the PR is approved like it had done before.

**Comment by denelon** [comment]
We're trying to make the bot a bit smarter, and it seems to be responding by becoming either slower or nondeterministic with it's responses.

**Comment by ImJoakim** [comment]
Worth a shot hehe. @denelon can you re-run it.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**
